### ISSUE TITLE
Options setIntroductionPath and setDescriptionPath are unavailable on command line

### ISSUE THREAD

**Issue by landong1973** [dgarijo/Widoco#267]
Options are required for automated process to generate documents for ontology. GUI is not really helpful in this case.

**Comment by dgarijo** [comment]
Yes, I got a bit lazy with this one. I don't know yet whether I should add it in the config file or explicit in console...
